### PR TITLE
Fix padding bottom and add max width to logo

### PR DESCRIPTION
**Pull request by gabrielgalc**
#### What is the purpose of this pull request?
Prevent user to upload giant logos.

#### What problem is this solving?
Prevent user to upload giant logos.

screen shot 2018-11-28 at 12 26 23 pm
screen shot 2018-11-28 at 12 26 38 pm

### CODE REVIEW

**Comment by gabrielgalc** [comment]
https://galc--delivery.myvtex.com/

**Comment by luciannojunior** [comment]
Nice!

File: react/components/TopMenu.js [hunk 1]
```diff
@@ -24,7 +24,11 @@ class TopMenu extends Component {
 
   renderLogo(mobileMode, logoUrl, logoTitle) {
     return (
+<<<<<<< HEAD
```
Comment (luciannojunior) [inline react/components/TopMenu.js]: Add CHANGELOG and fix merge conflicts

**Comment by gabrielgalc** [comment]
@luciannojunior

**Pull request merged**